utils/run_tests.sh

```diff
@@ -25,6 +26,9 @@ function wait_for_start {
     )
 }
 
-# depends_on in compose is not enough
-wait_for_start pystardog_stardog 5820
-pytest --endpoint http://pystardog_stardog:5820
+# will properly wait for cluster when working on implementing the cluster api calls.
+# also we need to point to an alb for a cluster.
+wait_for_start pystardog_stardog_1 5820
+sleep 50
```

Comment:
maybe you need `wait_for_start zoo1 2181` as well?

Reply:
I moved all cluster stuff related to its own branch. I will be addressing those comments in there.